generated/nidaqmx/_library_interpreter.py

```diff
@@ -390,6 +390,18 @@ def cfg_samp_clk_timing(
             task, source, rate, active_edge, sample_mode, samps_per_chan)
         self.check_for_error(error_code)
 
+    def cfg_time_start_trig(self, task, when, timescale):
+        cfunc = lib_importer.windll.DAQmxCfgTimeStartTrig
+        if cfunc.argtypes is None:
+            with cfunc.arglock:
+                if cfunc.argtypes is None:
+                    cfunc.argtypes = [
+                        lib_importer.task_handle, DateTime, ctypes.c_int]
```

Comment:
When should be an `AbsoluteTime`. `AbsoluteTime` is a ctypes type that should work correctly here, I think.